### PR TITLE
Implemented Pop up window for See Project Button click 

### PR DESCRIPTION
**Pull request by bobb-Rob**
### Implemented the following

- [ ] Use a JavaScript array to store all of the information for all projects.
- [ ] Projects section is created dynamically using the information stored in the JavaScript object.
- [ ] Popup with details about the project appears, when the user clicks (or taps) the button to check project details.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] Implement popups for both mobile and desktop screen sizes.

### CODE REVIEW

File: js_files/index.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                        <a href="${project.sourceFile}">See Source
+                        <img src="./icons/github-icon.svg" alt="github icon">
+                        </a>                        
+                    </button>
+                </div>
+            </div>
+        `;
+    projectModal.appendChild(popupContainer);
+    return projectModal;
+  };
```
Comment (JuliCarracedo) [inline js_files/index.js]: - [ ] Kindly make sure the popups look exactly like the ones in the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
review_image3

File: index.html [hunk 2]
```diff
@@ -1,317 +1,272 @@
...
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul>
+                <li class="tags">css</li>
+                <li class="tags">html</li>
+                <li class="tags">Bootstrap</li>
+                <li class="tags">Ruby</li>
+              </ul>
+              <button type="button" class="btn btn-feature">See Project</button>
```
Comment (JuliCarracedo) [inline index.html:124]: - [ ] Currently, this button is not working as intended. Kindly make sure that it causes a modal with data for that specific project to open

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @bobb-Rob , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

[✔️](https://emojipedia.org/check-mark/)No linter errors
[✔️](https://emojipedia.org/check-mark/)Professional README and repository
[✔️](https://emojipedia.org/check-mark/)JavaScript implementation working flawlessly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/popup.css [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  font-size: 2rem;
+  line-height: 2.75rem;
+  font-style: normal;
+  color: #172b4d;
+}
+
+.tech-stack {
+  display: flex;
+  margin-bottom: 1.325rem;
+}
```
Comment (ShahierNashaat) [inline css/popup.css]: - [ ] Please add flex-wrap: wrap; and gap: 10px; to make the design looks like the Figma design.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @bobb-Rob and @dunnyk , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bobb-Rob** [comment]
Thank you for the great review. I learn something new today about flex.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @bobb-Rob   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Excellent PR.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**